test/integration/controller/jobs/job/job_controller_test.go

```diff
@@ -2003,6 +2003,8 @@ var _ = ginkgo.Describe("Job controller interacting with scheduler when waitForP
 				g.Expect(createdJob.Status.Ready).ShouldNot(gomega.BeNil())
 				g.Expect(*createdJob.Status.Ready).Should(gomega.Equal(int32(1)))
 				g.Expect(k8sClient.Get(ctx, wlKey, wl)).Should(gomega.Succeed())
+				g.Expect(wl.Status.RequeueState).ShouldNot(gomega.BeNil())
```

Comment:
Please also verify that the `count` is not cleared

Reply:
Thanks. Added.